jwst/wfs_combine/wfs_combine.py

```diff
@@ -28,7 +21,8 @@ class DataSet:
 
     """
```

Comment:
For many functions, there are new parameters which should be added to the epydocs.

Reply:
done